### ISSUE TITLE
Encrypted source code

### ISSUE THREAD

**Issue by aemino** [equinusocio/vsc-material-theme#115]
Why is the source code of this repository encrypted?

**Comment by devsnek** [comment]
must be wannacrypt 😁

**Comment by equinusocio** [comment]
@aemino Only the theme builder is encrypted. Too many clones around that are been published under a different name and license. We decided to close the theme builder for everyone but contributors. The alternative was making the entire repository private loosing the issues tracking. 

PS: The file history still there so you can check the old theme builder version ant theme source.

**Comment by aemino** [comment]
It's understandable to have frustration about other people copying your work, I don't think that encrypting the primary source code of your repository is an equitable solution.  Frankly, I don't see a major issue for you on the front of others copying your work.  This package is popular enough to the point where clones have no chance of competing with this one.  This is a minor disadvantage that you exchange for the larger picture—allowing people to freely read the code of this **open source** repository and contribute.  Just my two cents.

**Comment by equinusocio** [comment]
This theme is open source. All theme files and icon files used bu vsc are available. The only crypted thing is the theme builder that is “stand-alone” and is not indispensable for cloning this project. I someone want to check the source code can simply open the relative json files :). If they want to collaborate they will be added as contributors so they can see the entire code. 

TL;DR: The only encrypted thing is the theme builder that is useful only if you want to collaborate and become a contributor.
